ops/opset13/relu.go

```diff
@@ -21,24 +21,9 @@ func (r *Relu) Init(_ []*onnx.AttributeProto) error {
 
 // Apply applies the relu operator.
 func (r *Relu) Apply(inputs []tensor.Tensor) ([]tensor.Tensor, error) {
-	t := inputs[0]
+	out, err := ops.ReLU(inputs[0])
```

Comment:
maybe check the err and return nil, instead of []tensor.Tensor{nil} if an error occurs?